.github/workflows/publish.yml

```diff
@@ -0,0 +1,126 @@
+name: build mreg-cli
+
+on:
+  push:
+    tags:
+      - mreg-cli-v*
+
+concurrency:
+  group: build-mreg-cli-${{ github.head_ref }}
+
+jobs:
+  build_pypi:
+    name: Build wheels and source distribution
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      # Only release from master branch
+      # https://stackoverflow.com/a/74690436
```

Comment:
If we create a branch, `notmaster`, that'd trigger it? Not that `/master` is appealing.

Reply:
Fixed in https://github.com/unioslo/mreg-cli/pull/254/commits/701ce7ac709910c26b12345d29e8bbaf42fed259

See the _Dump job github var_ step https://github.com/unioslo/mreg-cli/actions/runs/9445593618/job/26013521198 for other variables we can use to determine this...